specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1189 @@
+        "operationId" : "5ac4251d5b4ccd1554da7632",
+        "consumes" : [ "application/json", "text/json" ],
+        "produces" : [ "application/json", "text/json" ],
+        "parameters" : [ {
+          "name" : "modelVersion",
+          "in" : "query",
+          "description" : "(optional) This value indicates which model be used for scoring .",
+          "required" : false,
+          "type" : "string"
+        }, {
+          "name" : "showStats",
+          "in" : "query",
+          "description" : "(optional) if set to true, response will contain input and document level statistics.",
+          "required" : false,
+          "type" : "boolean"
+        }, {
+          "in" : "body",
+          "name" : "multiLanguageBatchInput",
+          "description" : "Collection of documents to analyze.",
+          "required" : false,
+          "schema" : {
+            "$ref" : "#/definitions/MultiLanguageBatchInput"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "A successful call results in a list of recognized entities returned for each valid document",
+            "examples" : {
+              "application/json" : {
+                "documents" : [ {
+                  "id" : "1",
+                  "entities" : [ {
+                    "text" : "100mg",
+                    "type" : "Organization",
+                    "offset" : 11,
+                    "length" : 5
+                  }, {
+                    "text" : "ibuprofen",
+                    "type" : "Chemical",
+                    "offset" : 17,
+                    "length" : 9
+                  }, {
+                    "text" : "twice daily",
+                    "type" : "Frequency",
+                    "offset" : 34,
+                    "length" : 11
+                  } ],
+                  "relations" : [ {
+                    "target" : "TBD",
+                    "source" : "TBD",
+                    "relationType" : "TBD"
+                  } ]
+                } ],
+                "errors" : [ ]
+              }
+            },
+            "schema" : {
+              "$ref" : "#/definitions/BioEntitiesBatchResult"
+            }
+          },
+          "500" : {
+            "description" : "Error Response",
+            "schema" : {
+              "$ref" : "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/entities/pii" : {
+      "post" : {
+        "summary" : "Entities from PII domain",
+        "description" : "The API returns a list of pii entities (\\\"SSN\\\", \\\"Bank Account\\\", \\\"Goverment ID\\\" etc) in a given document. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/text-analytics-supported-languages\">Supported languages in Text Analytics API</a> for the list of enabled languages.\n",
```

Comment:
add in the description what PII stands for: Personally Identifiable Information (PII) although it is fine if in the summary it is simplky the acronym